apps/mark-scan/backend/src/store.ts

```diff
@@ -25,10 +25,16 @@ const SchemaPath = join(__dirname, '../schema.sql');
  * Manages a data store for imported election definition and system settings
  */
 export class Store {
+  // We don't want PAT connection status to persist across machine restarts,
+  // but we want it to be accessed through the store instead of state machine context
```

Comment:
Why do we want that? Also, storing it like this means that the value will be lost when a backend restart happens (such as when reloading after a file changes in dev). Is that a problem?

Reply:
PAT devices are usually brought by the voter so we expect the device to be disconnected at the end of a voting session. In the event of a machine restart mid-session, we don't want to assume the same voter has returned to the same machine. It wouldn't be _bad_ to store PAT connection status in the db and update its value on restart, but it's unnecessary.

Re: backend restarts, it's a slight inconvenience (requires 1 extra screen + button click to skip calibration) but I wouldn't say it's worth it to keep this bit in the db for dev purposes only.

Reply:
Is this relevant anymore? It seems like we're not using this field now that it's just stored in the state machine context and it can just be removed